### ISSUE TITLE
tagObj is set in the global scope on decode

### ISSUE THREAD

**Issue by egerod** [themasch/node-ebml#24]
As the title says, tagObj is added to the global scope variable.
After doing some testing I'm pretty sure it's duo to a typo in the decoder readTag function.
```
//From /lib/ebml/decoder.js
EbmlDecoder.prototype.readTag = function() {

    debug('parsing tag');

    if (this._cursor >= this._buffer.length) {
        debug('waiting for more data');
        return false;
    }

    var start = this._total;
    var tag = tools.readVint(this._buffer, this._cursor);

    if (tag == null) {
        debug('waiting for more data');
        return false;
    }

    var tagStr = this._buffer.toString('hex', this._cursor, this._cursor + tag.length);

    this._cursor += tag.length;
    this._total += tag.length;
    this._state = STATE_SIZE;

    tagObj = {
        tag: tag.value,
        tagStr: tagStr,
        type: this.getSchemaInfo(tagStr).type,
        name: this.getSchemaInfo(tagStr).name,
        start: start,
        end: start + tag.length
    };

    this._tag_stack.push(tagObj);
    debug('read tag: ' + tagStr);

    return true;
};
```
Here tagObj is missing `var` causing tagObj to be set in the global scope.

**Comment by themasch** [comment]
Thanks, i'll fix it!

**Comment by themasch** [comment]
its fixed in version 2.2.2. Thanks for the report.
